### ISSUE TITLE
When running wget with output the Powershell gets cyrillic

### ISSUE THREAD

**Issue by Alienmaster** [PowerShell/PowerShell#15202]
## Steps to reproduce
connect with ssh to a linux (in my case a ubuntu 16.04LTS on a VM)
```powershell
ssh user@host
wget -O- https://ltdata1.informatik.uni-hamburg.de/bbb-live-subtitles/audio_fork.zip
```

## Expected behavior
Get download in stdout as gibberish
![grafik](https://user-images.githubusercontent.com/11733097/114304462-94ca6080-9ad3-11eb-9f81-4ad05a83a6ec.png)

## Actual behavior
changed the powershell charset to cyrillic. Not on the remote machine. If you disconnect the ssh connection the charset persist until you close the powershell and open a new one.

![grafik](https://user-images.githubusercontent.com/11733097/114304431-5af95a00-9ad3-11eb-87a9-9194fff04bcd.png)
## Environment data

Powershell is used with Windows Terminal 1.6.10571.0
```none
Name                           Value
----                           -----
PSVersion                      7.1.3
PSEdition                      Core
GitCommitId                    7.1.3
OS                             Microsoft Windows 10.0.19042
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Comment by mklement0** [comment]
Note that PowerShell's doesn't support raw byte output; among PowerShell commands, only .NET objects flow through the pipeline; when calling _external programs_, their stdout output is invariably interpreted as _text_, i.e. _decoded_ into .NET _strings_.

**Comment by iSazonov** [comment]
PowerShell can use SSH only as shadow transport in PowerShell remoting.

Previously MSFT team said that a scenario with explicit SSH connection is not currently supported.

**Comment by TravisEz13** [comment]
Remoting Working Group:
@Alienmaster You were in PowerShell then ran your repro steps inside that PowerShell session, correct?  You didn't use `Enter-PSSession`?

**Comment by Alienmaster** [comment]
Correct.

**Comment by PaulHigin** [comment]
WG-Remoting:
Duplicate of #15128
@Alienmaster  This has been fixed for 7.2 preview7 release.  Please test.
